keel-web/src/main/kotlin/com/netflix/spinnaker/config/KeelConfigurationFinalizer.kt

```diff
@@ -33,6 +34,14 @@ class KeelConfigurationFinalizer(
 
   private val log by lazy { LoggerFactory.getLogger(javaClass) }
 
+  // TODO: not sure if we can do this more dynamically
+  @PostConstruct
+  fun registerApiExtensionsWithObjectMappers() {
+    objectMappers.forEach {
+      it.registerKeelEc2ApiModule()
+    }
+  }
```

Comment:
When the app starts this registers the `keel-ec2-api` extensions with the object mappers

Reply:
Hmm, this should be dynamically determined based on the plugins available, no? Ties to discussion here: https://github.com/spinnaker/keel/pull/1347/files#r449252729